### PR TITLE
Add a clarification for autoscaling on minikube

### PR DESCRIPTION
**Pull request by morhook**
Added a very small clarification on how to run this guide on `minikube`.

### CODE REVIEW

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 1]
```diff
@@ -50,6 +50,12 @@ new kinds of resource that represent metric readings.
 To learn how to deploy the Metrics Server, see the
 [metrics-server documentation](https://github.com/kubernetes-sigs/metrics-server#deployment).
 
+For enabling on minikube you can enable it with
```
Comment (Shubham82) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:53]:
```suggestion
If you are running {{< glossary_tooltip term_id="minikube" >}}, run the following command to enable metrics-server:
```

**Review by Shubham82 — CHANGES_REQUESTED** [review]

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md (hunk 1 above)
Comment (Shubham82) [reply content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:53]: @morhook , Maybe adding something like this would be better.

**Comment by morhook** [comment]
> Welcome @morhook! It looks like this is your first PR to [kubernetes/website](https://github.com/kubernetes/website) 🎉. Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval. You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands). You can also check if kubernetes/website has [its own contribution guidelines](https://github.com/kubernetes/website/tree/master/CONTRIBUTING.md). You may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md) if you run into trouble with your tests not passing. If you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). Also, for tips and tricks in the contribution process you may want to read the [Kubernetes contributor cheat sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet/README.md). We want to make sure your contribution gets all the attention it needs! Thank you, and welcome to Kubernetes. 😃

thanks to you @k8s-ci-robot ! Love to read you from Argentina  (🇦🇷 championsof the world on soccer for 2 more years!). I've been working on k8s for some time now and would love to continue helping on this awesome community and technology.

**Comment by Shubham82** [comment]
Thanks @morhook
Preview looks good to me.
https://deploy-preview-43808--kubernetes-io-main-staging.netlify.app/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#before-you-begin

/lgtm

**Comment by Shubham82** [comment]
@morhook It is not the label problem, once the PR gets `lgtm` label, you need the `approve`  label to merge the PR, and the approver approves the PR.

See the above comment from the bot: https://github.com/kubernetes/website/pull/43808#issuecomment-1792772785

**Comment by morhook** [comment]
who can approve this PR?

**Comment by Shubham82** [comment]
cc @sftim 
If it seems good to you, please approve this PR.

Thanks

**Review by sftim — COMMENTED** [review]
Nice.

Thank you.

/approve

**Pull request merged**